site/gitops/README.md

```diff
@@ -0,0 +1,8 @@
+# Gitops
+
+Packages eventually have to be deployed into one or more clusters. The default
+deployment mechanism is [Config Sync](gitops/configsync/), but since configuration is stored in standard
+repositories, other gitops tools can also be used.
```

Comment:
GitOps (capitalization) -- here and everywhere

I also suggest linking to https://opengitops.dev/ on the first mention in the body of the text